plugin/src/main/java/net/dzikoysk/funnyguilds/data/database/Database.java

```diff
@@ -22,7 +22,7 @@ public Database() {
 
         this.dataSource.setMaximumPoolSize(poolSize);
         this.dataSource.setConnectionTimeout(c.connectionTimeout);
-        this.dataSource.setJdbcUrl("jdbc:mysql://" + c.hostname + ":" + c.port + "/" + c.database + "?useSSL=" + c.useSSL);
+        this.dataSource.setJdbcUrl("jdbc:mysql://" + c.hostname + ":" + c.port + "/" + c.database + "?useSSL=" + c.useSSL + "?characterEncoding=utf8mb4");
```

Comment:
During creating db scheme, FG does not declare character encoding for tables, so it uses a default one. If we'd enforce `utf8mb4` for all clients, it could be a breaking change. I think we should move this property to configuration and if `c.encoding` will be empty, we should not declare `?characterEncoding` parameter at all as it was before.

Reply:
Everyone uses `utf8mb4` by default, so it wouldn't do any break changes in database.

Reply:
```
 MySQL includes character set support that enables you to store data using a variety of character sets and perform comparisons according to a variety of collations. The default MySQL server character set and collation are latin1 and latin1_swedish_ci, but you can specify character sets at the server, database, table, column, and string literal levels. 
```

https://dev.mysql.com/doc/refman/5.7/en/charset.html

Reply:
Only MySQL 8.0+ uses utf8mb4 by default. The 5.x line is still strong and present in the market. 
> From MySQL 8.0, utf8mb4 is the default character set, and the default collation for utf8mb4 is utf8mb4_0900_ai_ci.